README.md

```diff
@@ -1 +1,200 @@
-# Hello_Microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div> -->
```

Comment:
- [ ] As mentioned by the previous reviewer, kindly remove this `div` tag and also `-->`
![Screenshot from 2023-07-07 08-31-35](https://github.com/Izebiafe/Hello_Microverse/assets/91142432/8f152347-85c6-47a3-b9be-acf9a0cad9e2)

> * [ ]  Please remove this closing `div` tag and also `-->` symbol because the starting `div` tag is not here so kindly remove them +1